### ISSUE TITLE
Adding the now deprecated relation "has dividend entity" from PATO

### ISSUE THREAD

**Issue by LarsVogt** [oborel/obo-relations#775]
For new term requests, please provide the following information:

## Preferred term label

has dividend entity

## Synonyms

has dividend

## Textual definition

A relationship between a ratio or proportion and its dividend.

## Suggested parent term

[RO](http://purl.obolibrary.org/obo/RO_0002502)

## Attribution

https://orcid.org/0000-0002-8280-0487

**Comment by LarsVogt** [comment]
This issue relates to the issue https://github.com/pato-ontology/pato/issues/454 in PATO. Nico told me to ask @anitacaron for help adding this.

**Comment by nataled** [comment]
c, or maybe even primary label ('dividend' brings to mind payment from investments, in case we want to avoid that).

**Comment by ddooley** [comment]
I'd second calling it "has numerator".  No need for word "entity" esp as that makes it sound like a class of thing that has a dividend.  Similarly for divisor -> "has denominator".  Divisor/dividend sound like they are part of a division process; whereas ratios can be expressed as numerator/denominator.  (Tho it seems they're all going out of style since 2010, related to math education?!  https://books.google.com/ngrams/graph?content=divisor%2Cdividend%2Cnumerator%2Cdenominator&year_start=1900&year_end=2019&corpus=en-US-2019&smoothing=0)

**Comment by LarsVogt** [comment]
"has numerator" and "has denominator" sound good. I am not a native speaker, so whatever labels you think fit best.

**Comment by rays22** [comment]
This relationship would be relevant to model phenotypic traits, for example, the traits that represent the ratio of urinary sodium to potassium concentration in biological samples, or the waist circumference to height ratio.

Similar relationships are used to model ratio-related plant phenotypic traits and attributes, e.g.[TO:0000278 root to shoot ratio](https://www.ebi.ac.uk/ols4/ontologies/to/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FTO_0000278) :
1. [TO:has_dividend_quality](https://www.ebi.ac.uk/ols4/ontologies/to/properties/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FTO_has_dividend_quality?lang=en)
2. [TO:has_divisor_quality](https://www.ebi.ac.uk/ols4/ontologies/to/properties/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FTO_has_divisor_quality?lang=en)

Regarding the label or synonyms, both `has numerator` and `has dividend` look fine to me.

Maybe also add a new `arithmetic relationship` to group this and any [related relationships.](https://github.com/oborel/obo-relations/issues/774)

**Comment by rays22** [comment]
## 2024-02-15 -- 2024-02-21
#### Since last Wednesday
* Reconsidered the use of PRO homology classes in [OBA -- #269](https://github.com/obophenotype/bio-attribute-ontology/issues/269#issuecomment-1948297858).
* [Added EFO-OBA protein level mapping terms](https://github.com/obophenotype/bio-attribute-ontology/issues/250#issuecomment-1948064472) to [efo_oba.sssom.](https://docs.google.com/spreadsheets/d/13qh7dLE38vMyz91oRqj6GzKjFohNazNKAJxKG6Plw1o/edit#gid=1005741851).
* Created a PR [#3227](https://github.com/obophenotype/uberon/pull/3227) to fix pancreas-related UBERON issue [#3208.](https://github.com/obophenotype/uberon/issues/3208).
* Reviewed UBERON PR [#3126](https://github.com/obophenotype/uberon/pull/3126#pullrequestreview-1890529394).
* Incremental progress on writing the uPheno manuscript.
* Investigated RO issues [#774](https://github.com/oborel/obo-relations/issues/774#issuecomment-1956460413) and [#775](https://github.com/oborel/obo-relations/issues/775#issuecomment-1956459138) relevant to [ratio-related traits and phenotypes](https://github.com/obophenotype/bio-attribute-ontology/issues/199)
#### Next
* Continue uPheno manuscript writing.
* Follow up RO issues [#774](https://github.com/oborel/obo-relations/issues/774#issuecomment-1956460413) and [#775](https://github.com/oborel/obo-relations/issues/775#issuecomment-1956459138) relevant to [ratio-related traits and phenotypes](https://github.com/obophenotype/bio-attribute-ontology/issues/199)
* [Map APO entities to OBA / GO](https://github.com/obophenotype/upheno/issues/928).
#### Blockers
* Need new RO arithmetic relationships requested in [#774](https://github.com/oborel/obo-relations/issues/774#issuecomment-1956460413) and [#775](https://github.com/oborel/obo-relations/issues/775#issuecomment-1956459138) to make progress on modelling ratio-related traits

**Comment by nataled** [comment]
Interestingly, going through some old work of mine that didn't progress, I created a placeholder relation 'has numerator' (and its counterpart, 'has denominator'). Can we use these terms as the preferred label? I did a google search:
```
numerator denominator: 54 million hits
dividend divisor:       5.3 million hits
```
Looks like the world prefers those labels too :)

Applies to #774 also.
